db/column_family.h

```diff
@@ -358,7 +358,7 @@ class ColumnFamilyData {
 
   std::atomic<int> refs_;      // outstanding references to ColumnFamilyData
   std::atomic<bool> initialized_;
-  bool dropped_;               // true if client dropped it
+  std::atomic<bool> dropped_;  // true if client dropped it
```

Comment:
i kinda feel acquire-release semantics would be more correct, but either way it's at least as good as it was before

Reply:
For compaction it only needs relaxed semantics. I didn't check the other call sites of IsDropped(). But as you said, it is not worse than it was before.